pkg/tcpip/stack/ndp.go

```diff
@@ -284,6 +300,12 @@ type NDPConfigurations struct {
 	// processed.
 	HandleRAs bool
 
+	// HandleRSs determines whether or not Router Advertisements will be
```

Comment:
Advertisements -> Solicitations. 

Also please outline that Router Solicitations will only be sent if the NIC is operating as a router.